<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>My_House_Inventory</class>
 <widget class="QWidget" name="My_House_Inventory">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/newPrefix/images/building.png</normaloff>:/newPrefix/images/building.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>580</y>
     <width>362</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="Date_Time_Title">
      <property name="text">
       <string>Today's Date/Time :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Date_Time_Title_Text">
      <property name="text">
       <string>Today's Date/Time:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="MushroomHouse">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/newPrefix/images/mushroom.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="CategoryTabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>711</width>
     <height>311</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::TabShape::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTableView" name="ItemTableFromJson"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Create_Item_Group_Box">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>227</width>
     <height>201</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>170</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>16</y>
      <width>205</width>
      <height>172</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="Create_An_Item_Button">
       <property name="text">
        <string>Create a new item</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/newPrefix/images/optimization.png</normaloff>:/newPrefix/images/optimization.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_item_name">
         <property name="text">
          <string>Item Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="5">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="Create_Item_Item_Name_Line_Edit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Create_Item_Quantity_Line_Edit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Create_Item_Date_Line_Edit">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Location_comboBox">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentText">
            <string>Fridge</string>
           </property>
           <property name="maxVisibleItems">
            <number>10</number>
           </property>
           <property name="maxCount">
            <number>2147483647</number>
           </property>
           <property name="minimumContentsLength">
            <number>1</number>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
           <property name="duplicatesEnabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Fridge</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cabinet</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yellow Box</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Black Stand</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Wall</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Category_comboBox">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>Fruits</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ingredients</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Snow Gear</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bathroom</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_quantity">
         <property name="text">
          <string>Quantity:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_date">
         <property name="text">
          <string>Date:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_location">
         <property name="text">
          <string>Location:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Category:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>227</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Category:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="Category_comboBox_2">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>Fruits</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ingredients</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Snow Gear</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bathroom</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_item_name_2">
          <property name="text">
           <string>Item Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="Create_Item_Item_Name_Line_Edit_2"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="Remove_Item_Button">
        <property name="text">
         <string>Remove item</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/newPrefix/images/subtract-of-shopping-cart.png</normaloff>:/newPrefix/images/subtract-of-shopping-cart.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="Create_Item_Group_Box_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>227</width>
     <height>201</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>170</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>16</y>
      <width>205</width>
      <height>172</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QPushButton" name="Adjust_Item_Button">
       <property name="text">
        <string>Adjust item</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/newPrefix/adjustment.png</normaloff>:/newPrefix/adjustment.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_item_name_3">
         <property name="text">
          <string>Item Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="5">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="Create_Item_Item_Name_Line_Edit_3"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Create_Item_Quantity_Line_Edit_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Create_Item_Date_Line_Edit_2">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Location_comboBox_2">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentText">
            <string>Fridge</string>
           </property>
           <property name="maxVisibleItems">
            <number>10</number>
           </property>
           <property name="maxCount">
            <number>2147483647</number>
           </property>
           <property name="minimumContentsLength">
            <number>1</number>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
           <property name="duplicatesEnabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Fridge</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cabinet</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yellow Box</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Black Stand</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Wall</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Category_comboBox_3">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>Fruits</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ingredients</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Snow Gear</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bathroom</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_quantity_2">
         <property name="text">
          <string>Quantity:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_date_2">
         <property name="text">
          <string>Date:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_location_2">
         <property name="text">
          <string>Location:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Category:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
